docs/userGuide/contentAuthoring.md

```diff
@@ -161,6 +161,22 @@ Each variable must have an id and the value can be any MarkBind-compliant code f
 
 In any file in your site, you can include the variable by surrounding the variable's id with double curly braces, e.g. <code>{<span></span>{options}}</code>
 
+Variables can refer to other variables that are declared earlier, including helpful variables provided by MarkBind.
+
+```html
+<!-- This is allowed, MarkBind helpful variables can be used. -->
+<span id="right_hand">{ {glyphicon_hand_right}}</span>
+
+<!-- This is allowed, the second variable will be assigned the contents of the first variable. -->
+<span id="first">This is the first variable.</span>
+<span id="second">{ {first}}</span>
+<span id="third">{ {second}}</span>
+
+<!-- This is forbidden, the fourth variable is not declared yet. -->
+<span id="third">{ {fourth}}</span>
+<span id="fourth">This is the fourth variable.</span>
+```
+

```

Comment:
Changed text so that additional space not required. Copy pasting the updated text into `diffChecker` has no issues as well

```
<tip-box>
<div>
This is allowed, MarkBind helpful variables can be used.<br>

`<span id="right_hand">`<code>{<span/>{glyphicon_hand_right}}</code>`</span>`
</div>
<div>
This is also allowed, the second variable will be assigned the contents of the first variable.

`<span id="first">This is the first variable.</span>`<br>
`<span id="second">`<code>{<span/>{first}}</code>`</span>`<br>
`<span id="third">`<code>{<span/>{second}}</code>`</span>`
</div>
<div>
This is forbidden, the fourth variable is not declared yet.

`<span id="third">`<code>{<span/>{fourth}}</code>`</span>`<br>
`<span id="fourth">This is the fourth variable.</span>`
</div>
</tip-box>
```
will be rendered as
![variable-tipbox](https://user-images.githubusercontent.com/31084833/40470992-d2cd3754-5f67-11e8-9901-3e08c28c3eb6.PNG)